### DISCUSSION TITLE
Duplicate entries to other sites (multi-site)

### DISCUSSION THREAD

**Discussion by wei2go** [craftcms/cms#10555] [Ideas] [answered]
It would be great if we can duplicate entries to a chosen site (in a multi-site system), if a section is set to "Only save entries to the site they are created in". In this way, editors can find it handy to have a template to start with, instead of creating all pages from scratch. This is particularly useful when the majority of the page content is constructed by fields like Neo/Super Table/Matrix, etc.

**Comment by wei2go** [comment, marked as answer]
Never mind. I realised it's not very practical, for example:

- Each site may have different sections available.
- If an entry has an "Entries" type field set to choose from a section which may be unavailable in the target site, how is it gonna work when duplicating the content?

There are probably other scenarios similar to this, which can make this feature very hard to build.

I feel like it's not worthy. Just let the editors create pages from scratch...

**Reply by brandonkelly** [reply]
This is why it hasn’t happened yet :) Not against adding it, but it will take some time to think through all the complexities.
